src/queries/update-profile-query.ts

```diff
@@ -1,12 +1,17 @@
-import {useMutation} from '@tanstack/react-query';
+import {useMutation, useQueryClient} from '@tanstack/react-query';
 import {CustomerProfileUpdate} from '@atb/api/types/profile';
 import {updateProfile} from '@atb/api';
 
 export const useProfileUpdateMutation = () => {
+  const queryClient = useQueryClient();
+
   return useMutation({
     mutationKey: ['patchProfile'],
     mutationFn: (profile: CustomerProfileUpdate) => {
       return updateProfile(profile);
     },
+    onSuccess: () => {
+      return queryClient.invalidateQueries({queryKey: ['getProfile']});
```

Comment:
I think the `getProfile` string should be as a constant somewhere, maybe exported from the `get-profile-query`.